l10n_it_reverse_charge/models/account_invoice.py

```diff
@@ -89,10 +89,15 @@ def rc_payment_vals(self, rc_type):
 
     def rc_credit_line_vals(self, journal):
         credit = debit = 0.0
+        rc_lines = self.invoice_line_ids.filtered(lambda l: l.rc)
+        rc_tax_ids = rc_lines.mapped('invoice_line_tax_ids')
+        amount_rc_tax = \
+            sum(self.currency_id.round(line.amount) for line in rc_tax_ids)
```

Comment:
@SimoRubi qui ```line``` è ```account.tax``` giusto? Quindi ```line.amount``` è l'aliquota dell'imposta, non l'imposta applicata alla riga della fattura, giusto?

Reply:
@eLBati sì, `line` è un `account.tax`, quindi `line.amount` è l'ammontare delle tasse per quella singola `account.tax`; con la somma ottengo l'ammontare delle tasse per tutte le `rc_lines`, che sono le righe della fattura aventi il flag `rc` abilitato.

Reply:
@eLBati ho capito cosa intendi, quel `line.amount` era effettivamente la percentuale, ho corretto in https://github.com/OCA/l10n-italy/pull/561/commits/c4f38f45465acd2866327a6f3087a0ef98e6b03a